AKS (PRIMES is in P)/akslb.md

```diff
@@ -0,0 +1,12 @@
++++
+reference = "Lemma 2.3 from ~ https://www3.nd.edu/%7eandyp/notes/AKS.pdf"
+state = "Draft"
+statement = """
+$e |- ( ph -> N e. ( ZZ>= ` ; ; ; ; 1 0 0 0 0 ) ) $.
+$e |- ( ph -> R e. ( 1 ... ( |^ ` ( 5 log N ) ) ) ) $.
```

Comment:
I don't think that one is right. Instead of 
```
( 5 log N ) 
```
we need 
```
( ( 2 log N ) ^ 5 )
```

Log in this paper is always the logarithm to the base 2.

Reply:
Right! I've read the $log^5 n$ wrong!

Reply:
To prove this bound we will also need a result about concave functions.
The following suffices.
Let $X= [a,b)$ be a half open interval and let $f: X \to \mathbb{R}$  and two times continously differentiable on $(a,b)$ such that $f^{\prime\prime} \leq 0$ for $x \in (a,b)$ then the following inequality holds for all $x\in X$
```math
f(x) \leq f(a)+(x-a) f^{\prime}(a)
```
The estimate follows by https://www.wolframalpha.com/input?i=derivative+of+lb%28x%29%5E%287%2F2%29+at+x%3D+10000 which shows that $f^{\prime}(10000) < 1$ and
the second derivative is less than 0 for $x > e^{\frac{5}{2}}$, see https://www.wolframalpha.com/input?i=second+derivative+of+lb%28x%29%5E%287%2F2%29 and $f(10000)<10000$ by 
https://www.wolframalpha.com/input?i=lb%2810000%29%5E%287%2F2%29